assets/files.wxs

```diff
@@ -1735,44 +1735,24 @@
             </Component>
           </Directory>
           <Directory Name="coreclr" Id="dirD84258F96F80409BA8CD117DCD52EC6E">
```

Comment:
Please also change the `Id` of the directory just to be safe that the MSI upgrades remove the old folders and replace it with the new ones

Reply:
Updated.